src/pages/DelegatoryValidator/StakingBar.tsx

```diff
@@ -29,33 +36,42 @@ export default function StakingBar({validator}: ValidatorStakingBarProps) {
   };
 
   return (
-    <ContentBox padding={4}>
+    <ContentBox>
       <ListItem>
-        <ListItem>
+        <ListItem
+          sx={{
```

Comment:
I think you can do something like
`sx={{sm:{"width":"30%"}}}`

Reply:
with `sx={{sm:{"width":"30%"}}}`, it will stretch the content to the whole width, instead of containing them in the 30% of the width